test/gurka/test_route_summary.cc

```diff
@@ -18,9 +18,10 @@ using namespace valhalla::baldr;
 using namespace valhalla::gurka;
 using namespace valhalla::mjolnir;
 
-namespace {
+TEST(TestRouteSummary, GetSummary) {
```

Comment:
Sorry this diff looks weird, I basically moved the map building step into the test.